config/credentials.yml.enc

```diff
@@ -1 +1 @@
-bSZHF6a1px4IDYW0xPmGsrbwYPruIzTyhUB6xXiEjD1y3BOLYXFeN+GsQJO1P2vUBor8K400gQpVRDr845bQDBhz+9a4+uPC8PHDj+XuWIFyRKBN7891TuSLJtwklZZfk5E9UoyW8LbQsG3GKC9C4S2NrFvNCTwrSSk2/CfTXcC6cI92RlBs0TiYPo4z29vS3pcj8Yqrt1QUPh2YJDPZUwZycy691gCQspMC1Bp488FK/Gdzs8Avz/pm7JCjbcUyRPXUmLbKq8dOjqPYdzpf9BGBamugX02/y5E+XwyUXjT5Lcjqn4zwbqv8RFVpB6Y9kkjUqrv+0QkjfFy/dOzzY/sa/itDnURJNizaKMmfPGe1Pnt9A5yssiDmOOZhmNdnXWS6h5ZtDBmFGluRKoW1VBqeE6Wc--uCQnF/VvUwGE+1sP--dmXXZckAQEBwVYvMCbTw3Q==
\ No newline at end of file
+9a34P6rlO2y1tBm134h72IlJ5g/3o1iN1bf7j4sidhZYaKy/sQq+me77YhqgjBGPJUBHtVMhEBKEsZKOLgHGn6XFCOMgkrDqOg/CFEBRTvI6acE0C/KPS71xqZAWb65uYypFe/qChGhiIXyw2QHG/88lGxU9Mii5dK3PcHR4jn4S9n1qlbPCJeS6QVotUNtvy5F0a5YEuXeSGSWW3n+vbWuriCMsRYdN4zDtHghw5ITL7IKiKNGn1GBYwF2+zUxTsLA34z+9Gu4lNujUN84GUNEK1px8aUGMnk5xjr5RbUD+VjFmfQ6Pk+ONz1wZN5F348JwuFh2FVKejz+dv1sw5p4UPDu7PNJ85TU5AYPbK2J6bGYq7sgqSYawWhKttZ4eUlxL9Sv+dt5s/RkzCVFXuIrTQYberSClquTaRTU08YEyNxvLn9qfBw1Ra2TpEgVMkjU/BEsOaWE0tLPdEexYQU8byVpgJoGmiVm3Gfa9zyHv+SwcmXQYFbkvTyU+Tp+qTsTIEYT6zNH4ISrm5pi6Ks7s1NMU33JehU2beUVnSHY7--ClTYQiOi1S5NimWi--o7znBCnbHDbuDhHuUZge6g==
```

Comment:
Add an empty line at the end of your file

Reply:
Fixed!

Reply:
The file contains encrypted API keys. It appears that adding an empty line prevents the application from properly decrypting it, resulting in an ActiveSupport::MessageEncryptor::InvalidMessage exception. Therefore, we had to remove that empty line.